tox.ini

```diff
@@ -40,6 +40,7 @@ ignore_outcome = True
 
 [testenv:docs]
 changedir = doc
+whitelist_externals=/usr/bin/make
```

Comment:
Should the Makefile contain a make target `doc` or `docs`? At the moment, one only can invoke the build of the documentation with `tox -e docs`.

Reply:
I agree with you on this issue and I already added it in #112. However, I am not sure, whether I would add this in this PR because currently it is more concerned with maintaining issues as pep8 is depreciated now